doc/changelog.rst

```diff
@@ -1,6 +1,27 @@
 Changelog
 =========
 
+2.0.0a0
+-------
+
+Breaking changes:
+* Added support for Python 3.12
+* Dropped support for Python 3.7
+* Flask-BabelEx is no longer supported; the package is no longer maintained and Flask-Babel is recommended/active instead.
+* Flask-Mongoengine is no longer supported due to that package being unmaintained.
+* Bootstrap2 and Bootstrap3 themes are no longer available.
+* All Flask-Admin config has been namespaced under `FLASK_ADMIN_`.
```

Comment:
* Remove the usage of the config `FLASK_ADMIN_SWATCH` (caused by `Admin()` instance's new parameter - `theme`)

Reply:
Good call, will tweak wording.